java/src/apps/ConfigBundle.java

```diff
@@ -22,7 +22,7 @@
  * @version $Revision: 17977 $
  * @since 3.3.1
  */
-public class ConfigBundle extends apps.Bundle {
+public class ConfigBundle extends jmri.Bundle {
```

Comment:
> On Feb 12, 2016, at 10:01, Bob Jacobsen <notifications@github.com> wrote:
> 
> In java/src/apps/ConfigBundle.java:
> 
> > @@ -22,7 +22,7 @@
> >   * @version $Revision: 17977 $
> >   * @since 3.3.1
> >   */
> > -public class ConfigBundle extends apps.Bundle {
> > +public class ConfigBundle extends jmri.Bundle {
> Note also that this change breaks the Startup Preferences panel.
> 
> Is there a (straightforward) way I can add a test to detect that in the future?
> 
You can test that *every* call to Bundle.getMessage() does not throw an error or return null. That's the most straightforward (and probably slowest) test. Unfortunately, I know of no way to maintain that either.